### ISSUE TITLE
Top banner section doesn't resize dynamically

### ISSUE THREAD

**Issue by ericjsilva** [oabsa/jamboree#1]
The top banner section does not resize dynamically when the browser is resized or breakpoint changes.

**Comment by pburtchaell** [comment]
Yeah the sizes are pulled in using `window.innerHeight`. It could be done with CSS, but I was too lazy to write the clever CSS for it. It can be resolved with this: https://developer.mozilla.org/en-US/docs/Web/Events/resize
